index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title></title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <div id="main">
+      <div id="intro">
```

Comment:
Instead of using the id main on div , I could have used main tag as it is

Reply:
This request lacks a "why". Can you please explain to the author why the change was requested?